Event/SendinblueListSubscriber.php

```diff
@@ -97,6 +106,11 @@ public function listSubscribe(FormSavePostEvent $event): void
                 $mailTemplateId = $field['options']['mailTemplateId'] ?? null;
                 /** @var int|null $listId */
                 $listId = $field['options']['listId'] ?? null;
+                $redirectLink = $field['options']['redirectLink'] ?? null;
+
+                if ($redirectLink && $this->linkProviderPool) {
```

Comment:
Move check for `$this->linkProviderPool` that way phpstan now:

```suggestion
                if ($redirectLink) {
```